src/commodore/index.ts

```diff
@@ -42,9 +43,10 @@ export const defaultExtraConfig = {
   ignoreValues: ['params'],
   // map facts to files
   factsMap: {} as any,
+  githubBaseUrl: "https://github.com/"
```

Comment:
Please add the new extra config parameter in https://github.com/projectsyn/commodore-renovate/blob/master/src/commodore/readme.md#extra-configuration-file